README.md

```diff
@@ -48,7 +56,7 @@ Is simply a service that read ingress manifest annotations, and talks to the Clo
 
 ### Ingress-Controller
 
-We use Nginx ingress-controller to expose services from within the cluster to the Internet. The ingress-controllers main job is to connect hostnames and K8s services, and terminate SSL traffic.
+We use Nginx ingress-controller to expose services from within the cluster to the Internet. The ingress-controllers main job is to connect hostnames and K8s services, and terminate SSL traffic. Port 22 is opened to support SSH cloning from our Gitlab instance.
```

Comment:
```suggestion
We use Nginx ingress-controller to expose services from within the cluster to the Internet. The ingress-controllers main job is to connect hostnames and K8s services, and terminate SSL traffic. Port 22 is opened to support SSH to and from our Gitlab instance.
```

Reply:
Is 'to and from' accurate? What we do is allow inbound traffic on port 22 and forward it to gitlab-shell

Reply:
OK